glob/logs/log.go

```diff
@@ -0,0 +1,83 @@
+package logs
```

Comment:
we have `utilities` folder could you please put the files from glob into there?

Reply:
OK, I move then.